content/en/docs/concepts/services-networking/service.md

```diff
@@ -125,7 +125,7 @@ spec:
     image: nginx:11.14.2
     ports:
       - containerPort: 80
-        name: http-web-service
+        name: http-web-svc
```

Comment:
Checked the API (not the official one which is not accurate). The name has some constraints. "The port name can be no more than 15 characters long and it can only contain only [-a-z0-9] and must contain at least one letter [a-z]. It must not start or end with a hyphen ('-'), nor contain adjacent hyphens."